nuxeo-drive-core/src/main/java/org/nuxeo/drive/service/impl/FileSystemItemFactoryWrapper.java

```diff
@@ -35,7 +35,7 @@
 
     protected String facet;
 
-    protected FileSystemItemFactory factory;
+    protected transient FileSystemItemFactory factory;
```

Comment:
For all these `transient` it would be better to just stop making the class `Serializable`. We have lots of similarly unneeded `Serializable` classes in the rest of the Nuxeo code, dating back from the EJB days.

Reply:
Removed `Serializable` when possible.